xtas/tasks/_stanford_ner.py

```diff
@@ -52,7 +52,7 @@ def tag(doc, format):
     # in s.sendall(text). E.g. presumably the euro-sign:
     # UnicodeEncodeError: 'ascii' codec can't encode character u'\u20ac' in
     # position 460: ordinal not in range(128)
-
```

Comment:
I don't understand this comment. There is no s.sendall() anywhere here, although there is one in _frog.py. nltk.word_tokenize() apparently returns unicode (i.e. wide characters) and the Java process accepts UTF-8, so it has to be encoded. If something called from _frog.py doesn't accept wide characters and needs conversion to UTF-8, then that should be done there (and it is).